main/src/library/MutDisjointSets.flix

```diff
@@ -61,7 +61,9 @@ mod MutDisjointSets {
     ///
     /// Updates `s` with the elements of the iterable `m`.
     ///
-    pub def makeSets(m: m[t], s: MutDisjointSets[t, r]): Unit \ r with Iterable[m], Order[t] = region rc {
+    /// TODO: with Order[Iterable.Elm[m]] missing
```

Comment:
I think we can remove the TODO, keep the ticket, and then if we make a fix we can search for `Elm[m] ~ elt`.

Reply:
Fixed - the comment was a stray from before Matthew supplied his fix that I forgot to delete.